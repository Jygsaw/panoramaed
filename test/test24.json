{
  "students": [
    { "name": "Bruce Wayne", "nicknames": ["Batman"] },
    { "name": "Peter Parker", "nicknames": ["Spidey", "Spiderman", "Spider-Man"] }
  ]
}
