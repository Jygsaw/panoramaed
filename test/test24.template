<html>
  <head>
    <title>Title</title>
  </head>
  <body>
    <h1>Header</h1>
    <* EACH students student *>
      <* student.name *>
      <* EACH student.nicknames nickname *>
        <p><* nickname *></p>
      <* ENDEACH *>
    <* ENDEACH *>
  </body>
</html>
