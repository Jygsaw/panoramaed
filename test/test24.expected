<html>
  <head>
    <title>Title</title>
  </head>
  <body>
    <h1>Header</h1>
      Bruce Wayne
        <p>Batman</p>
      Peter Parker
        <p>Spidey</p>
        <p>Spiderman</p>
        <p>Spider-Man</p>
  </body>
</html>
